### PR TITLE
chore: Enable post deployment tests 

### PR DESCRIPTION
**Pull request by sawood14012**
# Description

adds scripts that allow us to do continuous testing in app-interface 

- [X] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: integration-tests/Makefile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+e2e:
```
Comment (prashbnair) [inline integration-tests/Makefile:1]: what is the use of this file?

Comment (sawood14012) [reply integration-tests/Makefile:1]: just following the Makefile approach / the job will run` make e2e` command to run e2e

Comment (prashbnair) [reply integration-tests/Makefile:1]: This is for running locally and nothing to do with the CD pipeline, is that correct?

Comment (sawood14012) [reply integration-tests/Makefile:1]: app-sre method follows the make file approach so i did

Comment (sawood14012) [reply integration-tests/Makefile:1]: nop

> This is for running locally and nothing to do with the CD pipeline, is that correct?

no the job will trigger this command to run e2e on the pod ref: https://github.com/sawood14012/fabric8-analytics-common/blob/continious-testing/integration-tests/openshift/f8a-e2e-tests.yaml#L38

Comment (prashbnair) [reply integration-tests/Makefile:1]: You already specified an entrypoint in your docker image. Why do you need this then?

Comment (sawood14012) [reply integration-tests/Makefile:1]: let me remove it

Comment (sawood14012) [reply integration-tests/Makefile]: done @prashbnair

**Review by prashbnair — APPROVED** [review]

**Pull request merged**